### DISCUSSION TITLE
V1.7.1

### DISCUSSION THREAD

**Discussion by dojohnso** [dojohnso/OctoPrint-SpoolManager#7] [Announcements]
*If you would like to help test new changes or just see what's coming before everyone else, head to your Software Update panel and set the Release Channel to Release & Development.*

## [BugFix]
- n/a

## [Enhancement]
- n/a

## Counter
![downloaded](https://img.shields.io/github/downloads/dojohnso/OctoPrint-SpooManager/1.7.1/total)

### NOTE: this plugin has been abandoned by the original creator and adopted here by a new maintainer

**This plugin is under new management** and will focus on critical bug fixes to start. Please bear with me as I get acclimated to this new plugin. If you would like to support these new efforts, please consider buying me a coffee or two. Thank you!

<a href="https://www.buymeacoffee.com/djohnson.tech" target="_blank"><img src="https://djohnson.tech/images/white-button.png" width=300 /></a>

<hr /><em>This discussion was created from the release <a href='https://github.com/dojohnso/OctoPrint-SpoolManager/releases/tag/1.7.1'>V1.7.1</a>.</em>

**Comment by gheja** [comment]
I'm using this plugin for all my prints, thanks for adopting it, I really appreciate it! :)

**Reply by dojohnso** [reply]
Thanks for the comment! I'm glad I can help people get back into this one. I use it for everything as well.
